src/pages/Profile.jsx

```diff
@@ -113,10 +115,10 @@ const Profile = () => {
               color="primary"
               size="small"
               aria-label="Set Container Permissions Button"
-              startIcon={<SettingsIcon />}
-              onClick={() => setShowAclPermsDocContainerModal(true)}
+              startIcon={<ShieldIcon />}
```

Comment:
Similar to above, think the gear icon from `SettingsIcon` looks better for this case.